contrib/seldon/README.md

```diff
@@ -11,8 +13,74 @@ kustomize build seldon-core-operator/base | kubectl apply -n kubeflow -f -
 
 ## Updating
 
-This kustomize spec was created from the seldon-core-operator helm chart with:
+See [UPGRADING.md](UPGRADING.md)
```

Comment:
We'll need to rename this here to `UPGRADE.md`, since now it will redirect to a 404 page

Reply:
Good catch! I've updated now